config/throttle.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Throttle Failed Authentication Attempts
+    |--------------------------------------------------------------------------
+    |
+    | Setting to enable/disable the AuthManager's throttling mechanism.
+    |
+     */
+    'useThrottle' => true,
```

Comment:
@mjauvin Let's use `throttleEnabled` for the config name, it makes it more clear what is happening.
```suggestion
    'throttleEnabled' => true,
```